bin/test/cucumber.rb

```diff
@@ -44,21 +44,26 @@
 
     sim_version = RunLoop::Version.new("#{sim_major}.#{sim_minor}")
 
-    if ENV["JENKINS_HOME"]
+    if RunLoop::Environment.jenkins?
       devices = {
         :iphone7Plus => 'iPhone 7 Plus',
         :air => 'iPad Air 2',
         :iphoneSE => 'iPhone SE',
         :iphone7 => 'iPhone 7'
       }
+    elsif RunLoop::Environment.azurepipelines?
+      devices = {
```

Comment:
When running in the pipeline, `devices` should contain these:

```
iPhone SE
iPhone Xs
iPhone Xr
iPhone Xs Max
iPad Pro 10.5-inch
iPad Pro 9.7-inch
iPhone 8 Plus
iPhone 8
iPhone X
```

Reply:
@jmoody , could you please confirm that we should add all these devices for testing

Reply:
Yes.  Please add those devices.